src/main/java/taxi/model/Driver.java

```diff
@@ -6,6 +6,8 @@ public class Driver {
     private Long id;
     private String name;
     private String licenseNumber;
+    private String login;
+    private String password;
```

Comment:
You can add this fields to equals and hashCode

Reply:
you should add these fields to equals and hashCode, but not to toString method

Reply:
++))